config.yaml.sample

```diff
@@ -27,3 +27,8 @@ bitcoind:
 bitcoind_ctl:
   endpoint: http://localhost:8888/bitcoind
   suspending_duration: 
+
+server_port: :8011
+
+notification:
+  tx_callback_url: https://autonomy-wallet.bitmark.com
```

Comment:
If the name is `tx_callback_url`, the value should be `https://autonomy-wallet.bitmark.com/api/accounts/notification`.

Reply:
And I just realized that the URL should accept all kinds of notification. Probably `notification_url` with enough succinctness.